certified-connectors/Connect2All/apiDefinition.swagger.json

```diff
@@ -334,7 +334,12 @@
   "info": {
```

Comment:
@Joerio Please update the host url as per the update you have uploaded in the certification portal

Reply:
I changed the url to the azurewebsites url, but this isn't the production URL, and this isn't used in the version that is currently published on PowerApps.